chainer/function_node.py

```diff
@@ -875,6 +878,7 @@ def _backprop(outputs, inputs, grad_required, retain_grad, grads):
                 v = node.get_variable_or_none()
                 if v is not None:
                     v.grad_var = g
+                    v.loss_scale = loss_scale
```

Comment:
It's better to use a "private" name, like `v._loss_scale`, and I think it should also be added to the definition of `Variable`. It's enough to add a class attribute `_loss_scale = None`, which is used as the default value.

Reply:
OK, will change the variable name from `loss_scale` to `_loss_scale` and initialize it as you suggested.